tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyMain.java

```diff
@@ -1308,17 +1310,45 @@ public JSONObject statusJSON(HttpServletRequest request) throws JSONException {
 
         double load = osmxbean.getSystemLoadAverage();
         CLDRConfig config = CLDRConfig.getInstance();
+
+        String sessid = request.getParameter("s");
+        if (sessid == null) {
+            HttpSession hsession = request.getSession(false);
+            if (hsession != null) {
+                sessid = hsession.getId();
+            }
+        }
+        CookieSession mySession = null;
+        UserRegistry.User myUser = null;
+        if (sessid != null) {
+            mySession = CookieSession.retrieveWithoutTouch(sessid);
+        }
+        if (mySession == null) {
+            sessid = null;
+        } else {
+            sessid = mySession.id;
+            myUser = mySession.user;
+        }
+        String orgName = (myUser == null) ? null : myUser.getOrganization().getDisplayName();
+        JSONObject perm = (myUser == null) ? null : myUser.getPermissionsJson();
+
         return new JSONObject().put("isBusted", isBusted).put("lockOut", lockOut != null).put("isSetup", isSetup)
             .put("isUnofficial", isUnofficial()).put("environment", config.getEnvironment().name())
             .put("specialHeader", config.getProperty("CLDR_HEADER"))
             .put("specialTimerRemaining", specialTimer != 0 ? timeDiff(System.currentTimeMillis(), specialTimer) : null)
             .put("processing", startupThread.htmlStatus()).put("guests", CookieSession.getGuestCount())
             .put("users", CookieSession.getUserCount()).put("uptime", uptime).put("surveyRunningStamp", surveyRunningStamp.current())
             .put("memfree", free).put("memtotal", total).put("pages", pages).put("uptime", uptime).put("phase", phase())
-            .put("currev", SurveyMain.getCurrevCldrApps()) // Code only!
             .put("newVersion", newVersion).put("sysload", load).put("sysprocs", nProcs).put("dbopen", DBUtils.db_number_open)
             .put("dbused", DBUtils.db_number_used)
-            .put("contextPath", request.getContextPath());
+            .put("contextPath", request.getContextPath())
+            .put("isPhaseBeta", isPhaseBeta())
+            .put("sessionId", sessid)
+            .put("user", myUser)
+            .put("organizationName", orgName)
+            .put("permissions", perm)
+            .put("specialHeader", getSpecialHeaderText())
+        ;
     }
```

Comment:
This function statusJSON is now too large; it should have subroutines, maybe with a new class just for the items that go in the json